packages/driver/src/cypress/command_queue.ts

```diff
@@ -60,336 +60,322 @@ const commandRunningFailed = (Cypress, state, err) => {
   })
 }
 
-export default {
-  create: (state, timeout, whenStable, cleanup, fail, isCy) => {
-    const queue = $queue.create()
-
-    const { get, slice, at, reset, clear, stop } = queue
-
-    const logs = (filter) => {
-      let logs = _.flatten(_.invokeMap(queue.get(), 'get', 'logs'))
+export class CommandQueue extends Queue<Command> {
+  state: any
+  timeout: any
+  whenStable: any
+  cleanup: any
+  fail: any
+  isCy: any
+
+  constructor (state, timeout, whenStable, cleanup, fail, isCy) {
+    super()
+    this.state = state
+    this.timeout = timeout
+    this.whenStable = whenStable
+    this.cleanup = cleanup
+    this.fail = fail
+    this.isCy = isCy
+  }
 
-      if (filter) {
-        const matchesFilter = _.matches(filter)
+  logs (filter) {
+    let logs = _.flatten(_.invokeMap(this.get(), 'get', 'logs'))
 
-        logs = _.filter(logs, (log) => {
-          return matchesFilter(log.get())
-        })
-      }
+    if (filter) {
+      const matchesFilter = _.matches(filter)
 
-      return logs
-    }
-
-    const names = () => {
-      return _.invokeMap(queue.get(), 'get', 'name')
+      logs = _.filter(logs, (log) => {
+        return matchesFilter(log.get())
+      })
     }
 
-    const add = (command) => {
-      queue.add(command)
-    }
```

Comment:
Did this get dropped on accident?

Reply:
This can be deleted because we are using inheritance in the new code.